pkgs/applications/misc/kiwix/lib.nix

```diff
@@ -51,4 +51,12 @@ stdenv.mkDerivation rec {
   postPatch = ''
     patchShebangs scripts
   '';
+
+  meta = with lib; {
+    description = "Common code base for all Kiwix ports";
+    homepage = "https://kiwix.org";
+    license = licenses.gpl3;
```

Comment:
In their READMEs libkiwix and kiwix-tools state "GPLv3 or later" as the license.

Use `license = licenses.gpl3Plus;` to avoid ambiguity with `gpl3Only`.

https://github.com/kiwix/kiwix-tools#license
https://github.com/kiwix/libkiwix#license

Reply:
good catch! i've updated the licenses. kiwix-desktop (pkgs/applications/misc/kiwix/default.nix) is also gpl3+

<https://github.com/kiwix/kiwix-desktop#license>